tests/foreman/cli/test_partitiontable.py

```diff
@@ -86,7 +86,9 @@ def test_positive_create_with_content(self, module_target_sat):
         :CaseImportance: Critical
         """
         content = 'Fake ptable'
-        ptable = module_target_sat.cli_factory.make_partition_table({'content': content})
```

Comment:
@pondrejk `test_positive_create_with_content` and `test_positive_create_with_content_length` seem to be testing the same thing. Probably we can merge them and add a parameterized test instead. wdyt?

Reply:
on a second thought  `test_positive_create_with_content_length` no longer needs to be there, it made sense to check the length limit when the content was supplied directly. but now the file path is required so the possibility of 1270181 returning is close to none